### PR TITLE
Sign release commit

### PR DESCRIPTION
**Pull request by romcheg**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+ralph (20190109.2) bionic; urgency=medium
```
Comment (vi4m) [inline debian/changelog]: why this change? Do we want to issue new release at the same time?

Comment (romcheg) [reply debian/changelog]: Nope, as usual I pushed the test commit unintentionally. Currently I’m rushing the bus driver to get to my workplace and kick this commit ASAP :)

**Review by szymi- — APPROVED** [review]

**Review by MarekBleschke — APPROVED** [review]
